src/services/template/migrations/1605033627134-SeedCategory.ts

```diff
@@ -0,0 +1,11 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class SeedCategory1605033627134 implements MigrationInterface {
```

Comment:
SeedCategory? This would imply that we have a table in our database called `category` that you are trying to seed

Reply:
Went off a guys tutorial, still working on this